pkg/controller/controller.go

```diff
@@ -121,9 +121,10 @@ const (
 	snapshotAPIGroup = snapapi.GroupName       // "snapshot.storage.k8s.io"
 	pvcKind          = "PersistentVolumeClaim" // Native types don't require an API group
 
-	tokenPVNameKey       = "pv.name"
-	tokenPVCNameKey      = "pvc.name"
-	tokenPVCNameSpaceKey = "pvc.namespace"
+	tokenPVNameKey                 = "pv.name"
+	tokenPVCNameKey                = "pvc.name"
+	tokenPVCNameSpaceKey           = "pvc.namespace"
+	tokenPVAnnoationTemplatePrefix = "pvc.annotations"
```

Comment:
it's `tokenPVCAnnoationTemplatePrefix `

Reply:
thanks for pointing that out, fixed

Reply:
still not fixed? `tokenPVAnnoationTemplatePrefix`